sdk/formrecognizer/Azure.AI.FormRecognizer/src/TrainingOperation.cs

```diff
@@ -103,8 +103,6 @@ private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToke
                     ? await _serviceClient.GetCustomModelAsync(new Guid(Id), includeKeys: true, cancellationToken).ConfigureAwait(false)
                     : _serviceClient.GetCustomModel(new Guid(Id), includeKeys: true, cancellationToken);
 
-                // TODO: Handle correctly according to returned status code
-                // https://github.com/Azure/azure-sdk-for-net/issues/10386
```

Comment:
I'm wondering whether we should handle update.Value.Status == Failed the same way as in the other LROs.

Reply:
I think it might be a bit risky to do it in the training operation. I agree that's extremely confusing to get a "Model status: invalid" with no exceptions thrown (I got very confused over that when I was getting used to the service).

However, even if the operation fails, the model has been created. I feel like throwing a `RequestFailedException` would give users the impression that no changes were done to their resource. I think it's something worth discussing a bit more.

Reply:
Yes, let's understand the reasons this could fail and what that means for the state of the model on the service side.  Would you file an issue to track this for Preview 3?

Reply:
#11473